python/src/pyngraph/node_factory.cpp

```diff
@@ -41,26 +41,19 @@ class DictAttributeDeserializer : public ngraph::AttributeVisitor
     {
     }
 
-    virtual void on_attribute(const std::string& name, std::string& value) override
```

Comment:
Why did you remove the adapter for string?

Reply:
The `on_adapter` for string remains. Anything that turns into a string, like enums, goes through `on_adapter`.